bugbug/models/component.py

```diff
@@ -203,3 +204,45 @@ def is_meaningful(product, component):
 
     def get_feature_names(self):
         return self.extraction_pipeline.named_steps["union"].get_feature_names()
+
+    def check(self):
+        success = super().check()
+
+        # Check that the most meaningful product components stills have at
+        # least a bug in this component. If the check is failing that could
+        # means that:
+        # - A component has been renamed / removed
+        # - TODO: Complete this list
+
+        limit = 1
+
+        for product, component in self.meaningful_product_components:
+            query_data = [
+                # TODO: Do we want to match bugs in graveyard?
+                ("classification", "Client Software"),
+                ("classification", "Developer Infrastructure"),
+                ("classification", "Components"),
+                ("classification", "Server Software"),
+                ("classification", "Other"),
+                # Search bugs in the given product and component
+                ("product", product),
+                ("component", component),
+                # We just wants to check if at least one bug exists, we don't
+                # need to download all the bugs for every component
+                # ("count_only", 1), # TODO: Bugzilla class doesn't likes when
```

Comment:
You could add a new function to bugzilla.py to do this (using libmozdata.bugzilla).

Reply:
The `_download` function is really meant to download data for a set of bugs, here instead we want something different, so a new function would be better.

The function could be called `def get_count(query)`, where query can be a dict of params. In the function, you can either use libmozdata.bugzilla, or even requests directly (like `download_bugs_between` is doing).